crates/env/src/engine/on_chain/ext.rs

```diff
@@ -348,6 +348,11 @@ mod sys {
             output_ptr: Ptr32Mut<[u8]>,
             output_len_ptr: Ptr32Mut<u32>,
         );
+
+        pub fn seal_rent_status(
+            output_ptr: Ptr32Mut<[u8]>,
+            output_len_ptr: Ptr32Mut<u32>,
+        );
```

Comment:
This does not match the signature:
https://github.com/paritytech/substrate/blob/1cbf0e16832eca5e0733d65279872ce79ddfe9f2/frame/contracts/src/wasm/runtime.rs#L1612